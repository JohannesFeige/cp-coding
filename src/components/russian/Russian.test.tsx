import React from 'react';
import { shallow } from 'enzyme';
import Russian from './Russian';

describe('Russian', () => {
  it('should render', () => {
    const wrapper = shallow(<Russian />);
    expect(wrapper).toMatchSnapshot();
  });
});
