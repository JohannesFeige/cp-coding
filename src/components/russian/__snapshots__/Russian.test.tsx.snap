// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Russian should render 1`] = `
<div>
  <h3>
    Russian Peasant Multiplication
  </h3>
  <p>
    The algorithm of the Russian Peasant Multiplication works as follows: you divide the left number by two as long as you reach one. Decimal places get truncated. The right number is written next to the left one and is doubled each time. At the end you have a table with two columns. Each number from the right column is crossed out where the corresponding number on the left is even. The last step is to add all the numbers from the right column that is not crossed out.
  </p>
  <RussianPeasantMultiplication />
</div>
`;
